css/styles.css

```diff
@@ -386,3 +386,128 @@ li {
   margin-right: 2rem;
   color: #acb7c3;
 }
+
+/** FORM SECTION **/
+.form-section {
+  background-color: #6070ff;
+  border-top-left-radius: 10rem;
+  padding-top: 5rem;
+  position: relative;
+}
+
+.form-rectangle {
+  position: absolute;
+  width: 5.4rem;
+  height: 3.6rem;
+  top: 2rem;
+  right: 0;
+}
+
+.form-half-circle {
+  position: absolute;
+  width: 5rem;
+  height: 8.9rem;
+  top: 4rem;
+  right: 0;
+}
+
+.form-img1 {
+  position: absolute;
+  width: 2.1rem;
+  height: 2.1rem;
+  top: 5rem;
+  left: 15rem;
+}
+
+.form-heading {
+  text-align: center;
+  font-size: 4rem;
+  font-weight: 700;
+  color: #fff;
+  margin-top: 6rem;
+}
+
+.form-paragraph {
+  font-size: 2rem;
+  font-weight: 300;
+  text-align: center;
+  padding-top: 3rem;
+  line-height: 2.8rem;
+  color: #fff;
+}
+
+form {
+  margin-top: 2rem;
+}
+
+fieldset {
+  border: none;
+  margin-top: 2rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.text,
+.email,
+.text-area {
+  padding: 2rem 1rem;
+  font-size: 1.7rem;
+  color: #172b4d;
+  font-weight: 400;
+  display: block;
+  width: 100%;
+  border-radius: 1rem;
+  margin-bottom: 1.5rem;
+  border: none;
```

Comment:
- [OPTIONAL ] Kindly give 12px of padding to all placeholder or input value as you can see on the Figma design

Your design    |        Figma
---------:|:------------
![figmabad](https://user-images.githubusercontent.com/79058364/165413967-af46fde3-ba68-4327-95fd-93b83bb93fc3.png) | ![figmagood](https://user-images.githubusercontent.com/79058364/165414006-3f3100be-4317-4265-8153-c90b31a0a966.png)